https://issues.redhat.com/browse/ACM-8583 (#5863)
open-cluster-management/rhacm-docs: clusters/cluster_lifecycle/ansible_config_cluster.adoc

. If you want to initiate any Ansible jobs before the cluster is installed, select *Add an Automation template* in the _Pre-install Automation templates_ section.

. Select between a `Job template` or a `Workflow job template` in the modal that appears. You can also add `job_tags`, `skip_tags`, and workflow types.
+

> * Use the `Extra variables` field to pass data to the `AnsibleJob` resource in the form of `key=value` pairs. 

Reviewer: ```suggestion
* Use the *Extra variables* field to pass data to the `AnsibleJob` resource in the form of `key=value` pairs. 
```

Reviewer: IBM style has field names as bold, but we might not always use it that way. If you see this somewhere else in the file, please make it consistent.

> * Special keys `cluster_deployment` and `install_config` are passed automatically as extra variables, containing data on general information about the cluster and details about the cluster install configuration. 

Reviewer: ```suggestion
* Special keys `cluster_deployment` and `install_config` are passed automatically as extra variables. They contain general information about the cluster and details about the cluster installation configuration. 
```

Reviewer: Just a thought here to break this up.

> * Special keys `cluster_deployment` and `machine_pool` are passed automatically as extra variables, containing data on general information about the cluster and details about the worker nodes being created in the cluster.

Reviewer 2: Sorry @jc-berger I made the mistake of mentioning `machine_pool` it should be `install_config` instead.
Please change this line to:

* Special keys `cluster_deployment` and `install_config` are passed automatically as extra variables, containing data on general information about the cluster and details about the cluster install configuration.

Author: Thanks @fxiang1 ! I've applied your feedback. Would you please let me know if you think it looks good to merge now?
